es-core/src/resources/Font.cpp

```diff
@@ -600,12 +600,12 @@ float Font::getNewlineStartOffset(const std::string& text, const unsigned int& c
 			return 0;
 		case ALIGN_CENTER:
 			{
-				unsigned int endChar = (unsigned int)text.find('\n', charStart);
+				size_t endChar = (unsigned int)text.find('\n', charStart);
 				return (xLen - sizeText(text.substr(charStart, endChar != std::string::npos ? endChar - charStart : endChar)).x()) / 2.0f;
 			}
 		case ALIGN_RIGHT:
 			{
-				unsigned int endChar = (unsigned int)text.find('\n', charStart);
```

Comment:
/home/devel/temp/EmulationStation_Stable/es-core/src/resources/Font.cpp:609:60: warning: result of co
mparison of constant 18446744073709551615 with expression of type 'unsigned int' is always true [-Wta
utological-constant-out-of-range-compare]
                                return xLen - (sizeText(text.substr(charStart, endChar != std::string
::npos ? endChar - charStart : endChar)).x());
                                                                               ~~~~~~~ ^  ~~~~~~~~~~~
~~~~~~